vendor-extra/JatsContentBundle/src/EventListener/BuildView/FigGraphicFigureListener.php

```diff
@@ -7,15 +7,17 @@
 use DOMNodeList;
 use FluentDOM\DOM\Element;
 use Libero\ViewsBundle\Views\ConvertsChildren;
-use Libero\ViewsBundle\Views\OptionalTemplateListener;
+use Libero\ViewsBundle\Views\TemplateChoosingListener;
 use Libero\ViewsBundle\Views\TemplateView;
+use Libero\ViewsBundle\Views\ViewBuildingListener;
 use Libero\ViewsBundle\Views\ViewConverter;
 use function Libero\ViewsBundle\array_has_key;
 
 final class FigGraphicFigureListener
 {
     use ConvertsChildren;
-    use OptionalTemplateListener;
+    use TemplateChoosingListener;
+    use ViewBuildingListener;
```

Comment:
I was checking this as an example, and it does both the template choosing and the view building. Now looking for a case where only one happens.